processor/src/aux_table/mod.rs

```diff
@@ -24,8 +24,8 @@ mod tests;
 /// the `trace_len` of the bitwise coprocessor.
 /// - column 0: selector column with values set to ONE
 /// - column 1: selector column with values set to ZERO
-/// - columns 2-14: execution trace of bitwise coprocessor
-/// - columns 15-17: unused columns padded with ZERO
+/// - columns 2-15: execution trace of bitwise coprocessor
+/// - columns 16-17: unused columns padded with ZERO
```

Comment:
I know that this update makes sense given that you added 1 column, but I actually don't think this comment is correct. I think that column 17 is now the only unused column which gets padded, and columns 2-16 are used for the execution trace. This gives a bitwise coprocessor trace width of 15 and matches the code below where only column 17 is not added to the bitwise fragment. Can you double-check?

Reply:
Yeah, agreed. Column 17 is the only unused column which is getting padded in bitwise_pow2 co-processor. I've updated these comments. Please see [ffc21b0](https://github.com/maticnetwork/miden/pull/232/commits/ffc21b0b4b432435a6ec413db66ee37c4a4ee144).